arches/app/models/resource.py

```diff
@@ -292,13 +353,13 @@ def bulk_save(resources, transaction_id=None):
         se.bulk_index(documents)
         se.bulk_index(term_list)
 
-    def index(self, context=None):
+    def index(self, context=None, df=None):
```

Comment:
I don't think we need to pass in the DatatTypeFactory.  It is in fact a singleton, so we should just be able to instantiate it w/o penalty.

Also, at the end of this method I see a call to `save` method on the Resource super class.  That doesn't seem right to me.

Reply:
fair enough re datatypefactory - I've removed that.  re: the save method - that is part of a separate commit that was made some time (9+ months) ago, and I'm reluctant to remove it as part of this PR and without more testing about why it was added in the first place.

Reply:
Yeah - removing save from the index method will be a bit of work. I've created a ticket for it: https://github.com/archesproject/arches/issues/9278